src/modules/Checkbox/Checkbox.js

```diff
@@ -70,6 +58,15 @@ export default class Checkbox extends Component {
   }
 
   render() {
+    let type = this.props.type;
+
+    if (!type) {
+      type = 'checkbox';
+      if (_.includes(this.props.className, 'radio')) {
+        type = 'radio';
+      }
+    }
```

Comment:
Make sure Checkboxes get the proper default `type` attribute.  If no `type` was supplied, we infer it from the `className`.